Gems/Atom/RHI/Code/Source/RHI/ShaderResourceGroupData.cpp

```diff
@@ -42,6 +43,12 @@ namespace AZ
             return m_shaderResourceGroupLayout.get();
         }
 
+        ShaderResourceGroupBindless& ShaderResourceGroupData::GetBindless()
+        {
+            m_bindless.m_parent = this;
```

Comment:
Hopefully this will go away when this class is merged with SRGData. Setting this everything GetBindless is called seems wasteful.